src/app-layout/__tests__/trigger-button.test.tsx

```diff
@@ -549,9 +546,12 @@ describe('Visual Refresh Toolbar trigger-button', () => {
         ).toBe(false);
         expect(wrapper!.findByClassName(toolbarTriggerButtonStyles['trigger-tooltip'])).toBeNull();
         expect(() => getByText(mockTooltipText)).toThrow();
-        fireEvent.focus(wrapper!.getElement(), { relatedTarget: mockRelatedTarget });
+        fireEvent.focus(wrapper!.getElement());
 
-        expect(getByText(mockTooltipText)).toBeTruthy();
+        expect(() => getByText(mockTooltipText)).toThrow();
+        expect(
+          wrapper!.getElement().classList.contains(toolbarTriggerButtonStyles['trigger-wrapper-tooltip-visible'])
```

Comment:
What's the difference between this tooltip visibility check and previous ones, where you checked the presence of the tooltip's elements in the DOM tree?

Reply:
this is checking if the trigger wrapper is a wrapper of a trigger that should show tooltip. It helps determine where the logic breaks.